lib/vault/Dockerfile

```diff
@@ -2,44 +2,26 @@
 # base image
 FROM alpine:3.7
```

Comment:
Use alpine version from env or pin in docker-compose.yml

Reply:
fixed